.checkstyle

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Was this file supposed to be included?  It seems to be local to your development environment.